sdk/src/trace/always_off_sampler.cc

```diff
@@ -0,0 +1,26 @@
+#include "opentelemetry/sdk/trace/always_off_sampler.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace trace
+{
+namespace trace_api = opentelemetry::trace;
+
+SamplingResult AlwaysOffSampler::ShouldSample(
+    const SpanContext *parent_context,
+    trace_api::TraceId trace_id,
+    nostd::string_view name,
+    trace_api::SpanKind span_kind,
+    const nostd::span<AttributeKeyValue> &attributes) noexcept
+  {
+    return { Decision::NOT_RECORD, nullptr };
```

Comment:
Given the straightforward implementation, should this entire file be inlined into the header, in order to have fewer files overall and possibly benefit from inlining?

Reply:
Yeah, I talked with @ziqizh and we both think that makes sense. I've moved the functionality to the header.